modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartQuickAdd.js

```diff
@@ -139,6 +196,61 @@ export default function CartQuickAdd() {
 			else {
 				const {productConfiguration, skus, urls} = product;
 
+				const productInstance = skus[0];
+
+				if (productInstance.discontinued) {
```

Comment:
this may fall in an error, because you may have a discontinued product without replacement @alessiorendina fyi

Reply:
I will replace `discontinued` with `replacementSkuId` so that should eliminate that possibility.